universal-application-tool-0.0.1/app/auth/Authorizers.java

```diff
@@ -0,0 +1,36 @@
+package auth;
+
+/**
+ * This enum represents Authorizers to be used with pac4j.
+ *
+ * <p>The enum values should always look like &#60;Authorizer&#62;(Labels.&#60;Authorizer&#62;).
```

Comment:
Question: Is this (`&#60;` etc) necessary? Can we remove it? Also I am curious how you generate these.

Reply:
I just looked up how to get < and > characters in javadoc. So it'll render like `<Authorizer>(Labels.<Authorizer>)`. I should have used `&lt;` and `&gt;`!